include/deal.II/fe/fe_q_hierarchical.h

```diff
@@ -674,6 +704,16 @@ class FE_Q_Hierarchical : public FE_Poly<TensorProductPolynomials<dim>,dim>
   virtual std::pair<Table<2,bool>, std::vector<unsigned int> >
   get_constant_modes () const;
 
+  /**
+   * A function to interpolate scalar values, computed at the
+   * support points to the FE_Q_Hierarchical.
+   *
+   * This function is not implemented and throws an exception if called.
+   */
+  void interpolate(std::vector<double>       &local_dofs,
+                   const std::vector<double> &values) const;
```

Comment:
Except, of course, if it is virtual and the base class does something unreasonable (as I think you mentioned). In that case, please mark the function as `virtual` to make it clear that it is inherited.